members/models/activityparticipant.py

```diff
@@ -95,7 +102,7 @@ def payment_info(self, format_as_html: bool):
         except Exception:
             if format_as_html:
                 result_string = format_html(
-                    f"{html_error_pre}Andet er aftalt.{html_post} "
+                    f"{html_error_pre}Andet er aftalt {self.price_in_dkk}.{html_post} "
```

Comment:
Måske inkludere "kr" i teksten?
```suggestion
                    f"{html_error_pre}Andet er aftalt: {self.price_in_dkk} kr.{html_post} "
```